create_binary.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+set -x
+
+pushd build_calicoctl
+docker build -t calico-build .
+popd
+
+mkdir -p `pwd`/dist
+chmod 777 `pwd`/dist
+
+docker run -v `pwd`/:/code --name pyinstaller calico-build \
```

Comment:
If we want this code to be maintainable then it needs much better commenting.  If the code reviewer is having to ask these sort of questions then the code needs better commenting (or is wrong).